src/components/Create/components/Selection/SelectionCard.tsx

```diff
@@ -0,0 +1,95 @@
+import { Col, Divider, Row } from 'antd'
+import { ThemeContext } from 'contexts/themeContext'
+import { ReactNode, useCallback, useContext } from 'react'
+import { CheckedCircle, RadialBackgroundIcon } from './components'
+import { SelectionContext } from './Selection'
+
+const Border: React.FC<{ isSelected: boolean }> = ({
+  isSelected,
+  children,
+}) => {
+  const {
+    theme: { colors },
+  } = useContext(ThemeContext)
+
+  const borderColor = isSelected
+    ? colors.stroke.action.primary
+    : colors.stroke.tertiary
+
+  return (
+    <div
+      style={{
+        border: 'solid 1px',
+        borderColor,
+      }}
+    >
+      {children}
+    </div>
+  )
+}
+
+export interface SelectionCardProps {
+  name: string
+  title: ReactNode
+  icon?: ReactNode
+  titleBadge?: ReactNode
+  description?: ReactNode
+  isSelected?: boolean
+}
+
+export const SelectionCard: React.FC<SelectionCardProps> = ({
+  name,
+  title,
+  icon,
+  description,
+  children,
+}) => {
+  const { selection, setSelection } = useContext(SelectionContext)
+  const isSelected = selection === name
+
+  const onClick = useCallback(() => {
+    if (isSelected) {
+      setSelection?.(undefined)
+      return
+    }
+    setSelection?.(name)
+  }, [isSelected, name, setSelection])
+
+  return (
+    <Border isSelected={isSelected}>
+      <div
+        style={{ padding: '1.75rem 0', userSelect: 'none' }}
+        onClick={onClick}
+      >
```

Comment:
**suggestion** WDYT about adding a hover state to cards? Something like adding the `clickable-border` class is prolly fine for now.